includes/Form.class.php

```diff
@@ -165,6 +165,18 @@ function process($params) {
 		}
 		$email = $_POST["email"];
 		$phone = isset($_POST["phone"]) ? $_POST["phone"] : "";
+		$lists = (isset($_POST["nlbox"]) && $_POST["nlbox"]) ? $_POST["nlbox"] : array();
+		if (isset($_SESSION["image_random_value"])) {

```

Comment:
Yep - it's when you embed a subscription form that has a captcha field in it. The captcha image has a source to `ac_global/scripts/randomimage.php` which is what creates the session var. Then when we make the Ajax request to submit the form data (**to a local script, not AC's servers yet**) we have access to the same session data.